R/db.data.type.PrestoConnection.R

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2015-present, Facebook, Inc.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+
+#' @export
+db_data_type.PrestoConnection <- function(con, fields, ...) {
+  return(vapply(fields, function(field) dbDataType(con, field), character(1)))
```

Comment:
If I change it to be `character(0)` I get:
```
> db_data_type.PrestoConnection(con, 1)
Error in vapply(fields, function(field) dbDataType(con, field), character(0)) :
  values must be length 0,
 but FUN(X[[1]]) result is length 1
```

I'm just going to use `sapply` for simplicity